ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -53,14 +57,14 @@ jobs:
         timeout: 1800
         topology: *build
 
-  fedora-30/temp_commit:
+  fedora-30/test_sssd:
     requires: [fedora-30/build]
     priority: 50
     job:
-      class: RunPytest
+      class: RunADtests
```

Comment:
should be:
```
     class: RunADTests
```
with an uppercase T after AD. Otherwise PRCI crashes

Reply:
Thanks for comment